### PR TITLE
Insert Match v1

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Frontend/src/app/dropdown-player/dropdown-player.component.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  selector: 'app-dropdown-player',
+  templateUrl: './dropdown-player.component.html',
+  styleUrls: ['./dropdown-player.component.css']
+})
+export class DropdownPlayerComponent implements OnInit {
+
+	@Output() onSelectedPlayer: EventEmitter<PlayerDropdownSelected> = new EventEmitter();
+	@Input() name : string;
+
+	players;
```
Comment (lauroaraujo) [inline Frontend/src/app/dropdown-player/dropdown-player.component.ts]: Any reason you are not using types? (same to insert-match.component.ts)

File: Frontend/src/app/dropdown-player/dropdown-player.component.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	constructor(private playerService: PlayerService) { }
+
+	getDropdownValues(filter? : string) { 
+
+		this.playerService.getAll().subscribe(
+			(result) => {
+				this.players = result;
+				},
+			(error) => {
+				console.log(<any>error);
```
Comment (lauroaraujo) [inline Frontend/src/app/dropdown-player/dropdown-player.component.ts]: Why the `<any>` here? Same to insert-match.component.ts

File: Frontend/src/app/models/PlayerDropdownSelected.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export class PlayerDropdownSelected {
+	inputName;
```
Comment (lauroaraujo) [inline Frontend/src/app/models/PlayerDropdownSelected.ts]: Any reason you are not declaring the types here as well?

**Review by lauroaraujo — COMMENTED** [review]
I have missed previous PRs, so I don't know if you guys are just not using types at all, but I notice that you did not declare variable types anywhere... Other than that, lgtm

**Comment by ghost** [comment]
Addressed @lauroaraujo `s comments

**Pull request merged**